helpers/Process_Matrix/process_matrix.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.decomposition import PCA
+from helpers.Timers.timer import run_timer
+from typing import List
+import logging
+from concurrent.futures import ThreadPoolExecutor
+
+# from helpers.Eigne_Calculator.[filename] import [eig_func] as eigen_manual
+from helpers.Principal_Component_Calculator.pca import pca as pca_manual
+# from helpers.Singular_Value_Calculator.[filename] import [svd_func] as svd_manual 
```

Comment:
I like how you've stubbed in the other imports! this will make incorporating them later a snap!